x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/index.tsx

```diff
@@ -149,20 +62,12 @@ export const StatefulFieldsBrowserComponent: React.FC<FieldBrowserProps> = ({
 
       {show && (
         <FieldsBrowser
-          browserFields={browserFieldsWithDefaultCategory}
+          browserFields={browserFields}
           createFieldComponent={createFieldComponent}
           columnHeaders={columnHeaders}
-          filteredBrowserFields={
-            filteredBrowserFields != null ? filteredBrowserFields : browserFieldsWithDefaultCategory
-          }
-          isSearching={isSearching}
-          onCategorySelected={setSelectedCategoryId}
-          onHide={onHide}
-          onSearchInputChange={updateFilter}
           restoreFocusTo={customizeColumnsButtonRef}
-          searchInput={filterInput}
-          appliedFilterInput={appliedFilterInput}
-          selectedCategoryId={selectedCategoryId}
+          setShow={setShow}
+          show={show}
```

Comment:
Yea, it would just be default true in the component. Gonna leave it for now as to not restart CI in case something happens with the build and this can still make it in